workflow/controller/controller.go

```diff
@@ -199,6 +199,7 @@ var indexers = cache.Indexers{
 	indexes.CronWorkflowIndex:            indexes.MetaNamespaceLabelIndexFunc(common.LabelKeyCronWorkflow),
 	indexes.WorkflowTemplateIndex:        indexes.MetaNamespaceLabelIndexFunc(common.LabelKeyWorkflowTemplate),
 	indexes.SemaphoreConfigIndexName:     indexes.WorkflowSemaphoreKeysIndexFunc(),
+	indexes.WorkflowCompletedIndex:       indexes.MetaLabelIndexFunc(common.LabelKeyCompleted),
 	indexes.WorkflowPhaseIndex:           indexes.MetaWorkflowPhaseIndexFunc(),
```

Comment:
Can be found for phase index

Reply:
Will clean up and remove the `CompleteIndex`.